common/config/rush/browser-approved-packages.json

```diff
@@ -2,6 +2,10 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
   "packages": [
+    {
```

Comment:
Not a `browser-approved-package`. Can you move this to the non-browser file?

Reply:
Sure, makes sense. Done.